tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -923,10 +923,7 @@ def get_device_tree():
         out, err, rc = cmd.run()
         if not err:
             out = out.encode('utf8')
-            devlist = map(
-                lambda line: dict(zip(keys, line.split(' '))),
-                out.splitlines()
-            )
+            devlist = [dict(list(zip(keys, line.split(' ')))) for line in out.splitlines()]
```

Comment:
again, if you could add a comment, that would be better...

Reply:
Sure, thank you!